packages/nuxt/src/common/types.ts

```diff
@@ -8,6 +8,37 @@ export type SentryNuxtClientOptions = Omit<Parameters<typeof initVue>[0] & objec
 export type SentryNuxtServerOptions = Omit<Parameters<typeof initNode>[0] & object, 'app'>;
 
 type SourceMapsOptions = {
+  /**
+   * Suppresses all logs.
+   *
+   * @default false
+   */
+  silent?: SentryVitePluginOptions['silent'];
```

Comment:
We're not forwarding the `SentryVitePluginOptions` types in the other options because we want to be able to handle breaking changes of the bundler plugin packages internally without exposing them via the Nuxt SDK package. 

While we could change it later, when the breaking change actually occurs, let's use the actual types in this option and the ones below. This ensures that we don't accidentally break anything (e.g. if we'd forget to adjust them later). 

```suggestion
  silent?: boolean
```

Reply:
Sounds good, I changed the types of these parameters